CONTRIBUTING.md

```
@@ -0,0 +1,494 @@
+
+### Branching Strategy
+
+- `master` - Production-ready code
+- `develop` - Integration branch for features
+- Feature branches - Created from `develop` for new work
+
+### Commit Message Format
+
+We use [Conventional Commits](https://www.conventionalcommits.org/):
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)